src/pybind/mgr/dashboard_v2/.gitignore

```diff
@@ -9,6 +9,7 @@ __pycache__
 # IDE
 .vscode
 .idea
+*.egg
```

Comment:
Can you make this a little bit more specific? Like `*pycharm*.egg` or `*debug*.egg`?